module/purpose/Striker/data/config/Striker.yaml

```diff
@@ -1,2 +1,5 @@
 # Controls the minimum log level that NUClear log will display
-log_level: DEBUG
+log_level: INFO
+
+# Position on field to walk to (x,y,theta)
+ready_position: [4, 0, -3.14]
```

Comment:
my virtual season branch has config for each of the robots with each of the positions.

Reply:
Want me to merge in all the initial position stuff?

Reply:
might as well